### ISSUE TITLE
Can we prune and the quantize the target model at the same time?

### ISSUE THREAD

**Issue by zheng-ningxin** [Jangho-Kim/PSG-pytorch#2]
Hi thanks for the great work!
I'm very interested in you work and I'm curious that whether PSG can prune and quantize the target model at the same time? or it can only perform one of them(pruning and quantization) at the same time?

Looking forward to your reply, thanks!!

**Comment by Jangho-Kim** [comment]
Hello!  Thanks for the your attention!
PSG can prune and quantize at the same time because the target value of symmetric quantization contains a zero value
which means the pruning.  However, if you want to prune the model with specific target pruning ratio without an accuracy degradation, i think PSGD needs to be combined with a specific pruning method.

Thank you.

**Comment by zheng-ningxin** [comment]
Hi thanks for the reply! I notice that we can only specify the weight/activation bit for the model, can PSG support mix-precision quantization for different layers?

**Comment by Jangho-Kim** [comment]
Yes, it is possible because we modified the conv2d layer and it can be calculated with the scaling function of different bits per layer.
